packages/babel-helpers/src/helpers.js

```diff
@@ -461,6 +461,8 @@ helpers.set = template(`
 
       if (parent !== null) {
         set(parent, property, value, receiver);
+      } else {
+        receiver[property] = value;
```

Comment:
Hm... looks like this is not fine.

`receiver[property] = value` will execute setters defined in the `receiver`'s prototype, even though they should not be executed (the setter from `super` should be executed instead).